.github/workflows/main.yml

```diff
@@ -359,6 +344,104 @@ jobs:
           parallel: true
           path-to-lcov: ./coverage.lcov
 
+  notebook-matrix:
+    runs-on: ubuntu-latest
+    needs:
+      [
+        check-changes,
+        prepare-nonhindcast-cache,
+        prepare-wave-hindcast-cache,
+        prepare-wind-hindcast-cache,
+      ]
+    if: |
+      always() &&
+      (
```

Comment:
"always() &&" seems redundant

Reply:
I agree, however, I have become accustomed to this syntax in actions and you will find it throughout our tests. 

There is discussion here agreeing that you should not need to but that you do:
https://github.com/orgs/community/discussions/25789#discussioncomment-3249273